modules/users/client/config/users-admin.client.routes.js

```diff
@@ -16,6 +23,7 @@ angular.module('users.admin.routes').config(['$stateProvider',
         url: '/users/:userId',
         templateUrl: 'modules/users/client/views/admin/view-user.client.view.html',
         controller: 'UserController',
+        controllerAs: 'vm',
         resolve: {
           userResolve: ['$stateParams', 'Admin', function ($stateParams, Admin) {
             return Admin.get({
```

Comment:
Can we factor this out into a function, with how we're doing it in the Articles routes? And use it in the Edit route below as well.